lib/tests/streamlit/delta_generator_test.py

```diff
@@ -138,43 +139,6 @@ def test_sidebar_nonexistent_method(self):
             "Did you mean `st.echo()`?",
         )
 
-    @parameterized.expand(
```

Comment:
Why were these tests removed?

Reply:
I might be missing something, but I didn't see the value in a test that only asserts that the function signature of a few select widgets is fixed. It seemed a bit too brittle / not useful thing to validate to me, so I removed it when the test started failing due to the `st.checkbox` signature changing.

Happy to add the tests back with an explanatory comment if there's a good reason these were here.